ui/selectors/metamask-notifications/metamask-notifications.ts

```diff
@@ -1,16 +1,21 @@
+// @ts-check
 import { createSelector } from 'reselect';
-import { NotificationServicesController } from '@metamask/notification-services-controller';
+import {
+  NotificationServicesControllerState,
+  INotification as Notification,
+  TRIGGER_TYPES,
+  defaultState,
+} from '@metamask/notification-services-controller/notification-services';
 import { createDeepEqualSelector } from '../../../shared/modules/selectors/util';
 
-const { TRIGGER_TYPES } = NotificationServicesController.Constants;
-
-type Notification = NotificationServicesController.Types.INotification;
-
 type AppState = {
-  metamask: NotificationServicesController.NotificationServicesControllerState;
+  metamask: Partial<NotificationServicesControllerState>;
 };
 
-const getMetamask = (state: AppState) => state.metamask;
+const getMetamask = (state: AppState) => ({
+  ...defaultState,
+  ...state.metamask,
+});
```

Comment:
If selectors are called before engine is initialised, we can gracefully fill in properties with the controllers default state.